<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="anu-Latitude-3480" name="Assertions.SoftAssterions" tests="1" failures="1" timestamp="2021-02-05T08:54:31 IST" time="0.010" errors="0">
  <testcase name="softAss" time="0.010" classname="Assertions.SoftAssterions">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [XYZz] but found [XYZ],
	Actual isn&#039;t as expected expected [XxYZ] but found [XYZ],
	did not expect [ABC] but found [ABC],
	Not as Expected did not expect [ABC] but found [ABC],
	expected object to not be null,
	expected [true] but found [false],
	The condition is not true expected [true] but found [false],
	expected [false] but found [true],
	Condition is not false expected [false] but found [true],
	expected not same [Assertions.Rectangle@353d0772] but found [Assertions.Rectangle@353d0772]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [XYZz] but found [XYZ],
Actual isn't as expected expected [XxYZ] but found [XYZ],
did not expect [ABC] but found [ABC],
Not as Expected did not expect [ABC] but found [ABC],
expected object to not be null,
expected [true] but found [false],
The condition is not true expected [true] but found [false],
expected [false] but found [true],
Condition is not false expected [false] but found [true],
expected not same [Assertions.Rectangle@353d0772] but found [Assertions.Rectangle@353d0772]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at Assertions.SoftAssterions.softAss(SoftAssterions.java:35)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:134)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:597)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:816)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:766)
at org.testng.TestRunner.run(TestRunner.java:587)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:384)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:378)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:337)
at org.testng.SuiteRunner.run(SuiteRunner.java:286)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1109)
at org.testng.TestNG.runSuites(TestNG.java:1039)
at org.testng.TestNG.run(TestNG.java:1007)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- softAss -->
  <system-out/>
</testsuite> <!-- Assertions.SoftAssterions -->
